README.md

```diff
@@ -28,4 +28,48 @@ armor_monoid.register_armor_group("arcane", 100)
 <br/>
 As you can see, the argument is not a multiplier, but the base armor group
 rating. Calling this would mean players start off with an armor rating in
-"arcane" of 100 (no protection).
\ No newline at end of file
+"arcane" of 100 (no protection).
+
+Special armor groups
+====================
+
+Luanti defines a number of [special armor groups](https://github.com/luanti-org/luanti/blob/master/doc/lua_api.md#objectref-armor-groups)
+that have an engine-based effect and therefore must be handled uniquely by this
+monoid.
+
+`fall_damage_add_percent`
+-------------------------
+
+The armor monoid handles this group exactly like normal armor groups that have a
+base value of 100, but because its value is multiplicative with built-in fall
+damage, its final value is internally offset by -100 to respect this behavior.
+
+To grant a player fall damage reduction, use the `fall_damage_add_percent` group
+as you would any normal damage group:
+
+```
+armor_monoid.monoid:add_change(player,{fall_damage_add_percent=0.5},"mymod:half_fall_damage")
+```
+
+`immortal`
+----------
+
+The armor monoid treats any value of 1 or less as an indication that a player
+can suffer damage. Conversely, any value greater than 1 will make a player
+immune to all damage.
+
+To grant a player immortality, set this group to a value greater than 1 like so:
```

Comment:
Suggestion/idea: Lookup tables are more pleasant to read than block text. This would be more compact:

```
Value range used by `armor_monoid`:

 * <= 1: player can take damage
 * > 1: player is immune to all damage
```

Reply:
I like this idea. I went ahead and rewrote the values explanations for both special armor groups to list form which should hopefully make it more clear and easy to reference.